src/user-event/event-builder/common.ts

```diff
@@ -0,0 +1,48 @@
+export const CommonEventBuilder = {
```

Comment:
Very nice! One thing I'm not sure to get however is what would be considered common and what would not be?

Reply:
The others I see for now would be [TextInput specific ones](https://github.com/callstack/react-native-testing-library/pull/1396/files#diff-2a2277746640ea803ab6aee547f4e8e3c5050f82a860946497a768c8ee1c3293), which there are quite a few.

Reply:
Ok I see, seems good